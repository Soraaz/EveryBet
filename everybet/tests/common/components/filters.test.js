import React from "react";
import renderer from "react-test-renderer";

import Filters from "../../../src/common/components/filters.js";

describe("Filters", () => {
	describe("rendering", () => {
		it("Should render correctly", () => {
			const tree = renderer
				.create(
					<Filters
						categoryCallback={() => {}}
						orderCallback={() => {}}
					/>
				).toJSON();
			expect(tree).to.matchSnapshot();
		});
	});
});
