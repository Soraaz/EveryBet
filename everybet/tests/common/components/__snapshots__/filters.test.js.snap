// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filters rendering Should render correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#37597F",
      "flexDirection": "row",
      "height": 50,
      "justifyContent": "flex-start",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "flex-start",
        "marginHorizontal": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFFFFF",
        }
      }
    >
      Catégorie
    </Text>
    <View
      style={
        Object {
          "backgroundColor": "#FFFFFF",
          "borderRadius": 5,
          "height": 25,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
            "paddingLeft": 10,
          }
        }
      >
        <Text>
          Toutes
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "flex-start",
        "marginHorizontal": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFFFFF",
        }
      }
    >
      Ordre
    </Text>
    <View
      style={
        Object {
          "backgroundColor": "#FFFFFF",
          "borderRadius": 5,
          "height": 25,
          "justifyContent": "center",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
            "paddingLeft": 10,
          }
        }
      >
        <Text>
          Plus récents
        </Text>
      </View>
    </View>
  </View>
</View>
`;
